<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactAdder</class>
 <widget class="QWidget" name="ContactAdder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Url:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="last_name">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="notes">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Notes...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>First Name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Notes:</string>
   </property>
  </widget>
  <widget class="QLabel" name="newContact">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>45</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>New Contact</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="first_name">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="telephone">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Telephone</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Telephone:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tags:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="url">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>www.example.com</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>example@email.com</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>395</y>
     <width>291</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>E-mail:</string>
   </property>
  </widget>
  <zorder>label_5</zorder>
  <zorder>last_name</zorder>
  <zorder>notes</zorder>
  <zorder>label_1</zorder>
  <zorder>save_button</zorder>
  <zorder>label_2</zorder>
  <zorder>label_6</zorder>
  <zorder>newContact</zorder>
  <zorder>first_name</zorder>
  <zorder>back_button</zorder>
  <zorder>telephone</zorder>
  <zorder>label_3</zorder>
  <zorder>url</zorder>
  <zorder>email</zorder>
  <zorder>scrollArea</zorder>
  <zorder>label_4</zorder>
  <zorder>label_7</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
